app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands = command.split()
+    if commands[1] != commands[2]:
+        with open(commands[1], "r") as file_in,\
+                open(commands[2], "w") as file_out:
```

Comment:
what are commands[1], commands[2]?
create separated variables for them and call on demand